synapse/lib/lmdbslab.py

```diff
@@ -1030,6 +1037,20 @@ def last(self, db=None):
         finally:
             self._relXactForReading()
 
+    def lastkey(self, db=None):
+        '''
+        Return the last key/value pair from the given db.
+        '''
+        self._acqXactForReading()
+        realdb, dupsort = self.dbnames[db]
+        try:
+            with self.xact.cursor(db=realdb) as curs:
+                if not curs.last():
+                    return None
+                return curs.key()
```

Comment:
docstring does not match the behavior. either we return the last key / None, or we return the key+Value